be/src/storage/lake/persistent_index_memtable.cpp

```diff
@@ -16,82 +16,110 @@
 
 namespace starrocks::lake {
 
+void PersistentIndexMemtable::insert(const std::string& key, int64_t version, const IndexValue& value) {
+    std::list<IndexValueInfo> index_value_infos;
+    index_value_infos.emplace_front(version, value);
+    _map.emplace(key, index_value_infos);
+}
+
+void PersistentIndexMemtable::update(std::list<IndexValueInfo>& index_value_infos, int64_t version,
```

Comment:
```suggestion
void PersistentIndexMemtable::update(std::list<IndexValueInfo>* index_value_infos, int64_t version,
```
Normally we don't pass non-const reference in StarRocks.

Reply:
done